locust/test/test_parser.py

```diff
@@ -148,4 +148,12 @@ def _(parser, **kw):
         out.seek(0)
         stdout = out.read()
         self.assertIn("Custom boolean flag", stdout)
-        self.assertIn("Custom string arg", stdout)
\ No newline at end of file
+        self.assertIn("Custom string arg", stdout)
+
+    def test_csv_full_history_requires_csv(self):
```

Comment:
This is one of the few times you'll hear me saying it, but I think this is a test we dont actually need :)

Reply:
I can remove it, but I don't see why you would not like to test handling of invalid parameter combinations? You are never going to get full coverage either if you do not test this.

Reply:
Strict coverage metrics dont really say a lot. But leave it in if you like, I guess it doesnt hurt

Reply:
It does not say anything about the functionality (requirements correctness), but I disagree that it does not say as lot. It says a lot about the general code quality. If you have tested the functionality, then what does all the uncovered code do?
But let's close that discussion as it is a little off topic :)